hordak/admin.py

```diff
@@ -51,8 +52,12 @@ def get_queryset(self, *args, **kwargs):
             super()
             .get_queryset(*args, **kwargs)
             .annotate(
-                balance_sum=Sum("legs__amount"),
-                income=Sum("legs__amount", filter=Q(legs__amount__gt=0)),
+                # TODO: I think this needs some sign-changing work
```

Comment:
Agreed, something like `if obj.type` then flip the subtraction?

Reply:
Done